### DISCUSSION TITLE
Preparing for VulkanSceneGraph-1.1.8 - test result for Debian 12 on x86_64 with Intel(R) UHD Graphics 630

### DISCUSSION THREAD

**Discussion by reedev** [vsg-dev/VulkanSceneGraph#1282] [General]
Hi Robert,

I tested the vsgExamples and most of them are working ok (or with some lighting 'issues').

Machine:
* x86_64 (laptop)
* Intel(R) UHD Graphics 630 (CFL GT2)
* Debian bookwom (12)

Each example is running ok except:

vsgsubpass
  Segmentation fault

vsgrendertotexturearray models/teapot.vsgt
  Error: vsg::Device::create(...) failed to create logical device. VkResult = -7

vsgannotation models/openstreetmap.vsgt
  info: [Vulkan] Failed to find vkGetDeviceProcAddr in layer "libVkLayer_MESA_device_select.so"

vsgclip models/openstreetmap.vsgt
  Segmentation fault

vsgclip models/teapot.vsgt
  Funky rendering, correct?

vsgcomputevertex
  Segmentation fault

**Comment by xarray** [comment]
Intel i7-12650H, NVIDIA GTX 3050Ti
Windows 11, Visual Studio 16 2019 (x64).

vsgsubpass: working
vsgrendertotexturearray models/teapot.vsgt: working
vsgannotation models/openstreetmap.vsgt: working

vsgclip models/openstreetmap.vsgt: showing a black earth sphere

vsgclip models/teapot.vsgt: showing a black, clipped teapot

vsgcomputevertex: crashes at app/TransferTask.cpp <Line 73>: bufferInfo->data is nullptr

Other examples seem to be working fine.

**Comment by xarray** [comment]
I've made a small fix in app/TransferTask.cpp (Starting from Line 70), just check if data is empty, and vsgcomputevertex seems to be working now:

    for (auto& bufferInfo : bufferInfoList)
    {
        if (!bufferInfo->data)
        {
            continue;
        }

        std::string str;
        if (bufferInfo->data->getValue("name", str))
        ......
